addons/website_sale/static/src/js/location_selector/map/map.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <templates xml:space="preserve">
     <t t-name="website_sale.locationSelector.map">
-        <div class="w-100 h-100" style="width: 30rem! important; height: 30rem !important;" t-ref="mapContainer"></div>
+        <div class="w-100 flex-grow-1" t-ref="mapContainer"/>
```

Comment:
I don't think this would work, it's maybe why it fails in mobile... 

According to [leaflet quickstart guide](https://leafletjs.com/examples/quick-start/), the map container needs to have a defined height.

Reply:
I think the problem is that the map is rendered when the div is not shown, and then since we use bootstrap, I can't catch easily any event in javascript to inform the map that its size has been updated. 

I'll try to drop the use of tab-pane and do the switch in owl instead. If that doesn't work, I'll go back to good old js to catch the event